brave/browser-laptop
# Reorganize ledgerBackup.js

Fixes #11332
Closes #11343

- Remove obsolete styles
- Split l10n strings to force `font-weight: strong` (On Transifex HTML tags may be deleted by contributors, which should not happen)
- Set `cursor: text` to indicate where you can select the passphrase manually. Since the mouse cursor does not change you often see that the passphrase is not selected, while you though you did.
- Set the same styles as `syncTab.js` to avoid possible style inconsistency
- Split font shorthand to ensure that the styles could be cascaded properly. See: https://github.com/Khan/aphrodite#object-key-ordering

Auditors: @cezaraugusto

Test Plan:
1. Open about:preferences#payments
2. Enable Payments
3. Go to the advanced setting
4. Click `Backup your wallet`
5. Make sure that the cursor changes on the passphrase div

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
